src/components/primitives/Button/Button.js

```diff
@@ -1,7 +1,8 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-const getColor = variant => ({ primary: 'bg', secondary: 'primary' }[variant]);
+const getColor = variant =>
```

Comment:
Let's use `variant` here too at some point.

Reply:
I'll refactor tomorrow on the flight I think.